joint_trajectory_controller/doc/userdoc.rst

```diff
@@ -175,6 +175,9 @@ open_loop_control (boolean)
 
   Default: false
 
+allow_nonzero_velocity_stop (boolean)
```

Comment:
Proposal:
```suggestion
allow_nonzero_velocity_at_trajectory_end (boolean)
```

Current name sounds to me to have inverse logic, then the description says.

Reply:
You are right -> I pushed a commit renaming the parameter.